mscore/inspector/inspector_measurenumber.ui

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>InspectorMeasureNumber</class>
+ <widget class="QWidget" name="InspectorMeasureNumber">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>198</width>
+    <height>104</height>
+   </rect>
+  </property>
+  <property name="accessibleName">
+   <string>Chord Symbol Inspector</string>
```

Comment:
???

Reply:
Not adressed yet?

Reply:
Not yet :)
Today for sure

Reply:
unfortunately, still not ready... Will have to wait a bit more